docs/docs/protocol-specs/gas-and-fees/fee-payments-and-metering.md

```diff
@@ -197,18 +197,16 @@ sequenceDiagram
   Alice->>AccountContract: run entrypoint
   AccountContract->>FPA: enqueue FPA.pay_fee(max_fee) msg_sender == Alice as fee distribution function
   AccountContract->>App: app logic
-    App->>AccountContract:
+  App->>AccountContract: response
```

Comment:
@PhilWindle Mermaid diagrams without text descriptions in sequence diagrams aren't rendering properly since I moved them into the docs folder. I am not really sure why, so I added this placeholder text so that it is functional again. Since you drafted the rest of this doc, I'm checking with you to see if there is better text to have here than "response". I added "response" throughout the doc.